### ISSUE TITLE
Materializecss is no longer supported

### ISSUE THREAD

**Issue by les113** [Dogfalo/materialize#6615]
Don't waste your time.

They really should put a notice on materializecss.com to be upfront with users.

**Comment by johnknapp** [comment]
Yeah, wow, it's _awfully_ quiet around here. Sad too. I've used this in the past and liked it.

Any comments or updates on project status, @Dogfalo , please?

**Comment by Smankusors** [comment]
> Yeah, wow, it's _awfully_ quiet around here. Sad too. I've used this in the past and liked it.
> 
> Any comments or updates on project status, @Dogfalo , please?

see #6438

**Comment by Deep-Codes** [comment]
Seriously this very dead for a  large repo :(

**Comment by doughballs** [comment]
If anybody is interested - some long term contributors have forked the repo and are working their way through the major issues with a view to a new release very soon.

https://github.com/materializecss/materialize

Looks like they set up a new Gitter support channel too:

https://gitter.im/materializecss/materialize

**Comment by jshster** [comment]
Hi everyone, I would love to help get this moving again.  I'm using Materialize extensively.  Love working with it ... but it obviously has numerous flaws.  I'm not great with the whole Github setup (just never had it forced on me) and haven't been involved in Open Source previously.    If someone can get in touch and let me know how we get things moving - if we can get things moving, I'd be very keen.

**Comment by ChildishGiant** [comment]
We've now got nightly builds setup on the active fork, you can find them here https://nightly.link/materializecss/materialize/workflows/nightly/dev/build.zip

**Comment by slawek-kobiec** [comment]
I have managed to fix this issue by modifying source materialize.js file.
I have commented out lines to scaleX and scaleY in functions '_animateIn' and '_animateOut' in Dropdown block.
It fixes either nav bar dropdown issue and form select.


![sdf](https://user-images.githubusercontent.com/38790783/106384139-794f2500-63c1-11eb-9056-109d7b4ef280.png)

**Comment by leonardorafael** [comment]
After years using materializecss, I give up :(. Now I'm a contributor of another project called [beercss.com](https://www.beercss.com). Maybe you like it. I'm trying to spread it now and some feedbacks are welcome too.

![188708647-119ebc69-b1a4-41f4-8518-a5de7b1b2ed9](https://user-images.githubusercontent.com/12979016/201759881-c89d9878-4b8c-4248-9d92-f18c728bfa58.png)

**Comment by mxdpeep** [comment]
❤️🙋‍♂️

**Comment by Deep-Codes** [comment]
> After years using materializecss, I give up :(. Now I'm a contributor of another project called [beercss.com](https://www.beercss.com). Maybe you like it. I'm trying to spread it now and some feedbacks are welcome too.

Looks Great would love to use and Contribute!🎉

**Comment by Baraw** [comment]
> I have managed to fix this issue by modifying source materialize.js file.
> I have commented out lines to scaleX and scaleY in functions '_animateIn' and '_animateOut' in Dropdown block.
> It fixes either nav bar dropdown issue and form select.
> 
> ![sdf](https://user-images.githubusercontent.com/38790783/106384139-794f2500-63c1-11eb-9056-109d7b4ef280.png)

man, Thank you very much !!!

**Comment by ChildishGiant** [comment]
We'd welcome a PR over on the [new fork](https://github.com/materializecss/materialize) to fix this 😄 Speaking of, we released [1.1.0-alpha](https://github.com/materializecss/materialize/releases/tag/1.1.0-alpha) recently and would love feedback on it.

**Comment by Tronic247** [comment]
For those who are looking for a new framework see this https://tronic247.com/material/
Please help it instead of this : )

**Comment by sraboncmc** [comment]
> I have managed to fix this issue by modifying source materialize.js file. I have commented out lines to scaleX and scaleY in functions '_animateIn' and '_animateOut' in Dropdown block. It fixes either nav bar dropdown issue and form select.
> 
> ![sdf](https://user-images.githubusercontent.com/38790783/106384139-794f2500-63c1-11eb-9056-109d7b4ef280.png)

only remove scaleX and scaleY part from  "AnimateIn" sort out my problem, anyway thanks

**Comment by abalter** [comment]
https://github.com/materializecss/materialize
